examples/react/messenger/index.js

```diff
@@ -135,7 +141,30 @@ class MonkeyChat extends Component {
 	}
 	
 	handleConversationExit() {
-		
+		if(nextConversation){
```

Comment:
Can't we do this comparison inside `removeMemberFromGroup`? this way we avoid repeating code